applications/amathea/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # @globalfishingwatchapp/amathea
 
+## 0.14.9
```

Comment:
yes, it is automatically generated by the changesets... I'll create a branch with the project in case we need to recover something and remove it completely in another branch